src/components/dictionaryConcepts/components/ActionButtons.jsx

```diff
@@ -1,27 +1,36 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
+import AddMapping from './AddMapping';
 
 const ActionButtons = ({
   actionButtons, id, concept_class, showDeleteModal, version_url,
 }) => {
   const dictionaryPathName = localStorage.getItem('dictionaryPathName');
+  let showExtra;
+  if (actionButtons === true) {
+    showExtra = true;
+  }
 
   return (
     <React.Fragment>
-      {actionButtons === true && (
-      <Link
-        className="btn btn-sm mb-1 actionButtons"
-        to={`/edit/${concept_class}/${id}${dictionaryPathName}`}
-      >
-      Edit
-      </Link>
+      {showExtra && (
+        <React.Fragment>
+          <Link
+            className="btn btn-sm mb-1 actionButtons"
+            to={`/edit/${concept_class}/${id}${dictionaryPathName}`}
+          >
+          Edit
+          </Link>
+          <AddMapping />
+        </React.Fragment>
       )}
       <button
         type="button"
         className="btn btn-sm mb-1 actionButtons"
         id="retireConcept"
         onClick={() => { showDeleteModal(version_url); }}
+        style={{ marginLeft: '12px' }}
```

Comment:
@Duncanian Why are you adding styles here?

Reply:
@muhozi , let me refactor.